tests/monitors/elasticsearch_query/elasticsearch_query_test.py

```diff
@@ -0,0 +1,283 @@
+from contextlib import contextmanager
+from functools import partial as p
+
+import hashlib
+import json
+import pytest
+import requests
+import time
+
+from datetime import datetime
+from elasticsearch import Elasticsearch
+from random import randint
+
+from tests.helpers.agent import Agent
+from tests.helpers.assertions import has_datapoint_with_all_dims, has_datapoint_with_metric_name, http_status
+from tests.helpers.util import container_ip, run_service, wait_for
+
+VERSIONS = ["5.0.0", "5.5.0", "6.2.0", "6.4.2", "7.0.0", "7.5.1"]
```

Comment:
Let's just test the oldest supported and the newest.  Testing 6 versions is going to bog down the CI time and is kind of overkill.